config.default.yaml

```diff
@@ -154,7 +154,7 @@ electricity:
   custom_powerplants: false  #  "false" use only powerplantmatching (ppm) data, "merge" combines ppm and custom powerplants, "replace" use only custom powerplants
 
   conventional_carriers: [nuclear, oil, OCGT, CCGT, coal, lignite, geothermal, biomass]
-  renewable_carriers: [solar, onwind, offwind-ac, offwind-dc, hydro]
+  renewable_carriers: [solar, csp, onwind, offwind-ac, offwind-dc, hydro]
```

Comment:
I wonder if it's possible to use `csp-tower` everywhere, like we have different types of wind power? My feeling is that it would be more consistent.

Reply:
Done...rather than use csp-tower everywhere, I used just csp. When we have another technology different from csp-tower, then it will be easy to generalize them as csp.